### ISSUE TITLE
VCS and the alerting Date/Time

### ISSUE THREAD

**Issue by netniV** [WeblateOrg/weblate#2639]
**Is your feature request related to a problem? Please describe.**
When trying to diagnose issues that occurred, as long as you are on the current day, you are treated to seeing the time it occurred so you can find any appropriate logs that accompany the issue.  

**Describe the solution you'd like**
Have an option to display the true date/time of events for proper diagnosis

**Describe alternatives you've considered**
You can then hover over the date to see a UTC-based date/time but when you are trying to figure out what entries are relevant this can be time consuming and not very friendly as you have no way to even copy/paste the real dates that are appearing.

**Additional context**
For example, our VCS is currently constantly throwing up the following error;
```
fatal: 'origin' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists. (128)
```

But when you look at the alert screen to see what occurs, the alert has no date and time, just 4 hours ago so you have only a rough estimate.  When you view the repository maintenance section, it's shows dates in a similar format but also with general descriptions such as 'Yesterday', which do not really help in diagnosing issues.

![image](https://user-images.githubusercontent.com/9052188/54517406-7f68c380-4959-11e9-952e-edfa4ec3bed7.png)

![image](https://user-images.githubusercontent.com/9052188/54517438-9ad3ce80-4959-11e9-93d0-a576a11ac782.png)

**Comment by nijel** [comment]
The timestamp display is unified across Weblate to make it easy to read by users. In most cases the exact timestamp is not really important. For debugging it might be indeed useful, but the question is how to integrate such feature without polluting the UI for typical use case...

**Comment by netniV** [comment]
Yes, that was my thought.  Could it be a user based setting?  Since I am more likely wanting exact times, whereas the average user could just stay with the nicely formatted times (which i was happy with until I had a problem to disagnose)

**Comment by netniV** [comment]
I am thinking this could only apply where you know in the UI it's going to be displayed to the end user.
